### DISCUSSION TITLE
Is Flinkjobs compatible with GCP?

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#315] [Installation] [answered]
Hi

Our Flink jobs are failing in checkpointing while running on GCP cloud as the config map entries still have wasbs URLs for checkpoint paths. What are the steps to configure the flinkjob to checkpoint to GCP blobs.
@reshmi-nair
@AmiableAnil
@keshavprasadms

**Comment by beepdot** [comment, marked as answer]
For the obsrv BB, I can see that the changes are in progress. This should get completed as part 5.1.0 release.
Please follow this PR to track changes - https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/pull/1593
@anandp504  @SadanandGowda @manjudr

@AmiableAnil @reshmi-nair Are we planning to add the support for the Lern and Knowlg flink jobs as part of 5.1.0? Please see if we can plan for the same if not already done.

**Reply by vinukumar-vs** [reply]
@keshavprasadms @tsprasath For knowlg jobs, these changes are already done as part of release-5.2.0 of Knowlg-BB.
